packages/ui/src/components/shared/export/Json.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { AccountBackup } from '@coong/keyring/types';
+import { Download } from '@mui/icons-material';
+import { Button } from '@mui/material';
+import FileSaver from 'file-saver';
+import { ExportObject, Props } from 'types';
+
+interface JsonProps extends Props {
+  value: AccountBackup;
+  object: ExportObject;
+}
+
+export default function Json({ value, object }: JsonProps): JSX.Element {
+  const { t } = useTranslation();
+
+  const downloadJsonFile = () => {
+    const blob = new Blob([JSON.stringify(value, null, 2)], { type: 'text/plain;charset=utf-8' });
+    FileSaver.saveAs(blob, `coongwallet_${object.toLowerCase()}_backup_json_${Date.now()}.json`);
```

Comment:
In case this is an `AccountBackup`, I think we should include the account name in the file name, so it would be easier to recognize while file is for which account. The account name should be in lowercase, and be trimmed off all white space or replaced with underscore (`_`).

Similar in the download QR code image as well.

```jsx
FileSaver.saveAs(blob, `coongwallet_${object.toLowerCase()}_backup_${account_name}_${Date.now()}.json`);
```